src/components/DismissablePane.tsx

```diff
@@ -1,14 +1,19 @@
 import React, { Dispatch, SetStateAction } from 'react';
 import { Headline, Pane, PaneHeader } from '@folio/stripes/components';
+import { Button } from '@material-ui/core';
 
 interface DismissablePaneProps {
     displayed: boolean;
-    setDisplayed: Dispatch<SetStateAction<boolean>>;
```

Comment:
To test if this gets called, you can use a jest mock:

```tsx
const setDisplayed = jest.fn();
render(<DismissablePane displayed={...} setDisplayed={setDisplayed} />);
await userEvent.click(...);
expect(setDisplayed).toHaveBeenCalledWith(false);
```

If you want to test that it happens more than once (e.g. testing all close buttons in the same test), you can do `setDisplayed.mockReset()` to reset it's state.  More info: https://jestjs.io/docs/mock-function-api